dataprocessing/eda/test.py

```diff
@@ -0,0 +1,11 @@
+import viz
+import pandas as pd
+
+# sample data
+df_tmp = pd.read_csv('sample_data/titanic/train.csv')
+
+v1 = viz.vizualize(df_tmp)
+
+v1.dist_plot(col_tp='_all')
+v1.null_heat_plot()
+v1.class_pie_plot('Survived')
```

Comment:
add one extra line where you see ![11](https://user-images.githubusercontent.com/18291428/53345131-3b763600-393a-11e9-91bf-401b21db6b1a.png)  this